app/mailers/notifications_mailer.rb

```diff
@@ -0,0 +1,20 @@
+require 'open-uri'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.